contrib/proxy/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3'
+services:
+  proxy:
+    image: gopackages/proxy
+    depends_on:
+      - redis
+    ports:
+      - "3000:3000"
+    environment:
+      - "ATHENS_STORAGE_TYPE=disk"
+      - "ATHENS_DISK_STORAGE_ROOT=/usr"
+    networks:
+      - main
+  redis:
```

Comment:
Note to self: after #400 is merged, redis won't be needed unless it's configured

Reply:
Merged